.github/workflows/test-reporter.yml

```diff
@@ -11,10 +11,27 @@ on:
 jobs:
   report:
     runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.event == 'pull_request' }}
     steps:
       - uses: elastic/apm-pipeline-library/.github/actions/test-report@current
         with:
           artifact: test-results            # artifact name
           name: JUnit Tests                 # Name of the check run which will be created
-          path: "**/*.xml"                  # Path to test results (inside artifact .zip)
+          path: "**/elastic-serverless-forwarder-junit.xml"                  # Path to test results (inside artifact .zip)
           reporter: java-junit              # Format of test results
+          output-to: step-summary           # Write summary in the PR
+      - uses: dawidd6/action-download-artifact@v2
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          workflow: ${{ github.event.workflow_run.workflow_id }}
+          commit: ${{ github.event.workflow_run.head_commit.id }}
+          name: test-results
+      - uses: 5monkeys/cobertura-action@v13
```

Comment:
NIT: Use a pinned version if possible

Reply:
that's the pinned version https://github.com/5monkeys/cobertura-action/releases :)

Reply:
I meant the commit sha instead, https://github.com/5monkeys/cobertura-action/commit/4157521550018abbbd29dfdf1b5963e907ea37f0 🙏

Reply:
uhm. not a big deal. but why preferring the commit sha (and that specific one) instead of the tag? :)

Reply:
They recommend to use immutable sha commits in the [using-third-party-actions](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) section:

> Pin actions to a full length commit SHA
Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps mitigate the risk of a bad actor adding a backdoor to the action's repository, as they would need to generate a SHA-1 collision for a valid Git object payload. When selecting a SHA, you should verify it is from the action's repository and not a repository fork.

When possible we use that approach